myjs.js

```diff
@@ -0,0 +1,89 @@
+function getComputerChoice() {
+  let itemPicker = Math.floor(Math.random() * 3);
+  if (itemPicker === 0) {
+    return "ROCK";
+  } else if (itemPicker === 1) {
+    return "PAPPER";
+  } else if (itemPicker === 2) {
+    return "SCISSORS";
+  }
+}
+
+humanScore = 0;
+computerScore = 0;
+
+function playRound(playerSelection, computerSelection) {
+  if (playerSelection === "ROCK" && computerSelection === "SCISSORS") {
+    humanScore++;
+    numRounds++;
+    return "You Won! Rock beats Scissors";
+  } else if (computerSelection === "ROCK" && playerSelection === "SCISSORS") {
+    computerScore++;
+    numRounds++;
+    return "You Lose! Rock beats Scissors";
+  } else if (playerSelection === "PAPPER" && computerSelection === "ROCK") {
+    humanScore++;
+    numRounds++;
+    return "You Won! Paper beats Rock";
+  } else if (computerSelection === "PAPPER" && playerSelection === "ROCK") {
+    computerScore++;
+    numRounds++;
+    return "You Lose! Paper beats Rock";
+  } else if (playerSelection === "SCISSORS" && computerSelection === "PAPPER") {
+    humanScore++;
+    numRounds++;
+    return "You Won! Scissors beats Papper";
+  } else if (computerSelection === "SCISSORS" && playerSelection === "PAPPER") {
+    computerScore++;
+    numRounds++;
+    return "You Lose! Scissors beats Papper";
+  } else {
+    return playRound(playerSelection, getComputerChoice());
+  }
+}
+
+let numRounds = 0;
+const maxRounds = 5;
```

Comment:
if you want your variable to be accessible anywhere in the file you should be declaring them at the top so it's obvious every function has access to them, otherwise they should be in the lowest scope possible to do what they're needed for